paf-mvp-frontend/src/containers/welcome-widget/WelcomeWidget.tsx

```diff
@@ -12,53 +12,45 @@ import { Option } from '../../components/forms/option/Option';
 import { Tooltip } from '../../components/tooltip/Tooltip';
 import { SubPanel } from '../../components/sub-panel/SubPanel';
 import { OptionsGroup } from '../../components/forms/options-group/OptionsGroup';
-import { globalEventManager } from '../../managers/event-manager';
 import { getCookieValue } from '../../utils/cookie';
 import { Cookies } from '@core/cookies';
 import { fromClientCookieValues } from '@core/operator-client-commons';
-
-interface WelcomeWidgetProps {
-  brandName: string;
-  brandLogoUrl: string;
+import { Arrow } from '../../components/svg/arrow/Arrow';
+import { Refresh } from '../../components/svg/refresh/Refresh';
+import { NotificationEnum } from '../../enums/notification.enum';
+import { notificationService } from '../../services/notification.service';
+
+export interface IWelcomeWidgetProps {
+  brandName?: string;
+  brandLogoUrl?: string;
+  emitConsent?: (value: boolean) => void;
+  destroy?: () => void;
 }
 
-export const WelcomeWidget = (props: WelcomeWidgetProps) => {
+export const WelcomeWidget = ({ emitConsent, destroy }: IWelcomeWidgetProps) => {
   const [isOpen, setIsOpen] = useState(true);
   const [isDetailsPanelOpen, setIsDetailsPanelOpen] = useState(false);
 
-  const pafCookies = fromClientCookieValues(getCookieValue(Cookies.identifiers), getCookieValue(Cookies.preferences));
+  const pafCookies = fromClientCookieValues(
```

Comment:
I think now it would be safer to use `getIdsAndPreferences()` so that you get filtered data (handling the special text values).
WDYT?

Reply:
Indeed. After updating I have this function.